### PR TITLE
UND-759 Navigation, Desktop Mega Menu

### PR DESCRIPTION
**Pull request by Ashish-srij**
UND-759 Navigation, Desktop Mega Menu

### CODE REVIEW

File: stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss [hunk 1]
```diff
@@ -7,10 +7,13 @@
...
   min-height: 37.5rem;
-  padding: $spacing-64 $spacing-24 0 $spacing-48;
+  padding: $spacing-64 0 0 $spacing-48;
   position: absolute;
   top: 7.125rem;
-  z-index: 2;
+  z-index: 1;
+  max-width: $width-1440;
+  left: 50%;
+  transform: translateX(-50%);
```
Comment (surbhi90) [inline stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss:16]: @Ashish-srij can you use mixin for transform

File: stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss [hunk 2]
```diff
@@ -69,19 +69,18 @@
...
+            height: 1.125rem;
+            width: 1.25rem;
+            content: "";
           }
-          &:hover{
-            &::after{
-              background: url(#{$img-path-Iconfourback}/external-hover.svg) no-repeat 0 bottom/$spacing-32 $spacing-32;
+          &:hover {
+            &::after {
+              background: url(#{$img-path-Iconfourback}/external-hover.svg) no-repeat 0 bottom/1.25rem 1.125rem;
```
Comment (surbhi90) [inline stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss:83]: @Ashish-srij can we create the placeholder for similar code and use it here

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss (hunk 2 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss:83]: no-repeat 0 bottom/1.25rem 1.125rem;
@surbhi90 Only this part is common on above lines. Do you want me to make placeholder for this only.

Comment (surbhi90) [reply stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss:83]: @Ashish-srij Above code in 'subexternal' class like width, height, align-items looks similar to me so we can create the placeholder for that code

Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss:75]: @surbhi90 Done.

File: stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss (hunk 1 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss]: @surbhi90 Done.

**Review by surbhi90 — APPROVED** [review]

**Pull request merged**